readme.md

```diff
@@ -0,0 +1,57 @@
+## Connecting a repository
+
+In order to connect a repository with blog articles, you have to link it via `app:add-blog`. You also have to add a
+commit hook in your repository with the secret provided. This makes sure that both the initial creation and additional
+updates in the repository will sync it to the app.
+
+
+### Repository Layout
+
+The main file you need is the `blog.yml`, this contains the settings of your blog. An example below:
```

Comment:
You originally used blog.y**a**ml, but I think I prefer blog.yml (without the a). Maybe we should support both (or at least for migration)